nearlib/test/integration.test.js

```diff
@@ -43,6 +49,37 @@ test('create account and then view account returns the created account', async (
     fail('exceeded number of retries for viewing account');
 });
 
+test('deploy contract and make function calls', async () => {
+    var data = [...fs.readFileSync('../tests/hello.wasm')];  
+    const initialAccount = await account.viewAccount(aliceAccountName);
+    var deployResult = await nearjs.deployContract(
+        aliceAccountName,
+        "test_contract",
+        data,
+        "FTEov54o3JFxgnrouLNo2uferbvkU7fHDJvt7ohJNpZY");
+    waitForNonceToIncrease(initialAccount);
+    const args = {
+        "name": "trex"
+    };
+    var viewFunctionResult = await nearjs.callViewFunction(
+        aliceAccountName,
+        "test_contract",
+        "near_func_hello", // this is the function defined in hello.wasm file that we are calling
+        args);
+    expect(viewFunctionResult).toEqual("hello trex");
+});
+
+const waitForNonceToIncrease = async (initialAccount) => {
+    for (i = 0; i < TEST_MAX_RETRIES; i++) {
+        try {
+            const viewAccountResponse = await account.viewAccount(initialAccount['account_id']);
+            //console.log(viewAccountResponse);
```

Comment:
nit: remove

Reply:
Not sure what the comment is about. If it's about the try-catch : we still need it. There are socket hangups that happen all the time and make rpc calls quite flaky. (it'd be a fun task for someone to debug this since we now have a good repro case by removing this try catch)

Reply:
there is commented out code. Let's not have commented out code.